app/views/participants/validations/eligible.html.erb

```diff
@@ -0,0 +1,24 @@
+<% title = "You're eligible for this programme" %>
+<% content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_panel title_text: title, text: nil, classes: "govuk-!-margin-bottom-8" %>
+    <h2 class="govuk-heading-m">What happens next</h2>
+    <% if school_cohort.cip? %>
+      <p class="govuk-body">We’ll email you a link to access your materials within the next 24 hours.</p>
+    <% elsif school_cohort.fip? %>
+      <p class="govuk-body">You will not need to use this service again during your training.</p>
+      <% if school_cohort.delivery_partner %>
+        <p class="govuk-body">
+          <%= school_cohort.school.name %> has chosen <%= school_cohort.delivery_partner.name %> and <% school_cohort.lead_provider.name %> to deliver your training programme.
```

Comment:
In the review app, user 'fip-mentor@...' only included a delivery partner, not a lead provider. The 'and' still appeared however. Is it possible to code this so the 'and' is removed if only the delivery partner or lead provider name is present?

Reply:
That was a typo on my side, now fixed.